api/src/main/java/org/pmiops/workbench/cohortreview/ReviewTabQueryBuilder.java

```diff
@@ -112,7 +111,7 @@
   public QueryJobConfiguration buildQuery(Long participantId,
                                           DomainType domain,
                                           PageRequest pageRequest) {
-    String tableName = DomainType.MASTER.equals(domain)
+    String tableName = DomainType.ALLEVENTS.equals(domain)
```

Comment:
Which just leaves MASTER_TABLE, but I'll leave that up to you.

Reply:
Done.